pyproject.toml

```diff
@@ -64,7 +65,6 @@ ignore_errors = true
 
 [[tool.mypy.overrides]]
 module = [
-    "tests.*",
     "test.*",
```

Comment:
Maybe we could remove this from the [PTB template](https://github.com/exasol/python-toolbox/blob/main/project-template/{{cookiecutter.repo_name}}/pyproject.toml#L72)?